recipes/python-lzo/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+export CPPFLAGS="-I$PREFIX/include -I$PREFIX/include/lzo $CPPFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+
+$PYTHON setup.py install
```

Comment:
I guess that the package does not use `setuptools` then. You can probably remove it from the build section later.